<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherChartWidget</class>
 <widget class="QWidget" name="WeatherChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e7f4fb;
    color: #2e3c4d;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
}

/* 按钮样式 */
QPushButton {
    background-color: #a4d3ee;
    border: 1px solid #87ceeb;
    color: white;
    border-radius: 6px;
    padding: 5px 12px;
}
QPushButton:hover {
    background-color: #87ceeb;
}
QPushButton:pressed {
    background-color: #6dbde0;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_mainpage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>主页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_high_temp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>最高气温</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_lowtemp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>最低气温</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_windscale">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>风力</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_humidity">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>湿度</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
